src/services/db/GitFileSystemService.ts

```diff
@@ -333,6 +333,12 @@ export default class GitFileSystemService {
         )
 
         if (error instanceof GitError) {
+          if (
+            error.message.includes("fatal: path") &&
+            error.message.includes("exists on disk, but not in 'HEAD'")
```

Comment:
not a blocker for now, because we can raise a PR to improve this + bug is fixed here but i wonder if `error.message` is a robust enough property for this?

Reply:
seems like best effort, since this is directly reliant on the error message that git returns us
do have an alternative in mind?